specification/network/resource-manager/Microsoft.Network/2017-06-01/expressRouteConnector.json

```diff
@@ -0,0 +1,372 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+		"version": "2017-06-01",
+		"title": "NetworkManagementClient"
+	},
+	"host": "management.azure.com",
+	"schemes":[
+		"https"
+	],
+	"consumes": [
+		"application/json"
+	],
+	"produces": [
+		"application/json"
+	],
+	"security":[
+		{
+			"azure_auth": [
+				"user_impersonation"
+			]
+		}
+	],
+	"securityDefinitions": {
+		"azure_auth": {
+			"type":"oauth2",
+			"authorizationUrl":"https://login.microsoftonline.com/common/oauth2/authorize",
+			"flow":"implicit",
+			"description":"Azure Active Directory OAuth2 Flow",
+			"scopes": {
+				"user_impersonation":"impersonate your user account"
+			}
+		}
+	},
+	"paths":{
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/expressRouteConnectors/{expressRouteConnectorName}": {
+			"delete": {
+				"tags": [
+					"expressRouteConnectors"
+				],
+				"operationId": "expressRouteConnector_Delete",
+				"description": "Deletes the specified Express Route Connector Resource.",
+				"x-ms-examples": {
+                    "ExpressRouteConnectorDelete": { "$ref": "./examples/ExpressRouteConnectorDelete.json" }
+                },
+				"parameters": [
+					{
+						"$ref": "#/parameters/ResourceGroupNameParameter"
+					},
+					{
+						"$ref": "#/parameters/ExpressRouteConnectorNameParameter"
+					},
+					{
+						"$ref": "#/parameters/ApiVersionParameter"
+					},
+					{
+						"$ref": "#/parameters/SubscriptionIdParameter"
+					}
+				],
+				"responses": {
```

Comment:
The error response is denoted by ```"default"```. please [refer](https://github.com/Azure/azure-rest-api-specs/blob/2fb9a0b3b902335ff0b0033711c234431931ec9d/specification/apimanagement/resource-manager/Microsoft.ApiManagement/2016-10-10/apimapis.json#L81)
@ravbhatnagar please confirm